pom.xml

```diff
@@ -272,25 +276,53 @@
 					<excludes>
 						<exclude>**/*StaticTests.java</exclude>
 					</excludes>
+                                        <trimStackTrace>false</trimStackTrace>
+                                        <skipTests>${skip.surefire.tests}</skipTests>
 				</configuration>
-				<executions>
-					<execution>
-						<id>integration-test</id>
-						<goals>
-							<goal>test</goal>
-						</goals>
-						<phase>integration-test</phase>
-						<configuration>
-							<excludes>
-								<exclude>none</exclude>
-							</excludes>
-							<includes>
-								<include>**/*StaticTests.java</include>
-							</includes>
-						</configuration>
-					</execution>
-				</executions>
 			</plugin>
+                        <plugin>
+                                <groupId>org.apache.maven.plugins</groupId>
+                                <artifactId>maven-failsafe-plugin</artifactId>
+                                <version>${maven-failsafe-plugin.version}</version>
+                                <executions>
+                                        <execution>
+                                                <goals>
+                                                        <goal>integration-test</goal>
+                                                        <goal>verify</goal>
+                                                </goals>
+                                                <configuration>
+                                                        <includes>
+                                                                <include>**/IT*.java</include>
+                                                                <include>**/*IT.java</include>
+                                                                <include>**/*StaticTest.java</include>
```

Comment:
Why this one is not named as the others ITs?

Reply:
It should be `StaticTests` but I missed "s" in the end. I assume this class was created to put some static tests but currently there is only one test, should it be named as BridgeStartupIT or just StaticIT?

Reply:
renamed it to StaticIT to be consistent